virttest/vt_utils/sys_time.py

```diff
@@ -0,0 +1,61 @@
+#
+# library for time related helper functions
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; specifically version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat (c) 2023 and Avocado contributors
+# Author: Houqi Zuo <hzuo@redhat.com>
+import re
+import os
+
+from avocado.core import exceptions
+from avocado.utils import process
+
+
+def get_timezone():
+    """
+    Get system timezone.
+
+    :return: timezone name and timezone code
+    :rtype: dict
+    :raises: AttributeError or IndexError will be raised if the result is
+             NOT matched on the platform. Or, the OSError will be raised if
+             this function is NOT available.
+    """
+    # for Linux system
+    if os.name == "posix":
+        timezone_cmd = 'timedatectl | grep "Time zone"'
+        timezone_re = '^(?:\s+Time zone:\s)(\w+\/\S+|UTC)(?:\s\(\S+,\s)([+|-]\d{4})\)$'
+    else:
+        raise OSError("The function was NOT available on this system yet!")
+
+    timezone = process.run(timezone_cmd, shell=True).stdout_text
```

Comment:
Note that we might hit a CmdError exception here(ignore_status=False by default if I remember it correctly)